packages/gatsby-theme-carbon/src/components/Code/Code.module.scss

```diff
@@ -11,13 +11,17 @@
     position: relative;
     display: inline;
     padding: 0 0.5em;
-    font-size: 0.75em;
+    font-size: 0.875em;
     bottom: 0.0625em;
     border-radius: 4px;
     background-color: $ui-03;
     color: $text-01;
   }
 
+  td > code {
+    font-size: 12px;
+  }
+
```

Comment:
Without this it was computing to 12.25px instead of 12px. Has something to do with the specificity in the page-table scss file.

Reply:
I think It was computing to 12.25 because it was using em. So when the font size goes down to 14, .875*14 = 12.25